pkg/controller/controller_test.go

```diff
@@ -1093,6 +1094,32 @@ func TestProvision(t *testing.T) {
 				}
 			},
 		},
+		"provision with size 0": {
+			volOpts: controller.VolumeOptions{
+				PersistentVolumeReclaimPolicy: v1.PersistentVolumeReclaimDelete,
+				PVName:                        "test-name",
+				PVC:                           createFakePVC(requestedBytes),
+				Parameters: map[string]string{
+					"fstype": "ext3",
+				},
+			},
+			volWithZeroCap: true,
+			expectedPVSpec: &pvSpec{
+				Name:          "test-testi",
+				ReclaimPolicy: v1.PersistentVolumeReclaimDelete,
+				Capacity: v1.ResourceList{
+					v1.ResourceName(v1.ResourceStorage): bytesToGiQuantity(0),
```

Comment:
I'm unsure if returning 0 bytes in the PV is going to be helpful.  PVC won't bind to it because the capacity is not >= the requested.

Should we return size == PVC request in this case?  cc @jsafrane

Reply:
Yes, Seems 0 bytes is not a valid size here.  Apiserver will not create PV obj with size 0 . If csi client return CreateVolumeResponse with size 0, Maybe we can reset to PVC requested size